erpnext/docs/user/manual/en/setting-up/workflows.md

```diff
@@ -71,6 +79,23 @@ When the Leave Approver opens the Leave Application page, he can finally "Approv
 
 <img class="screenshot" alt="Workflow" src="{{docs_base_url}}/assets/img/setup/workflow-5.png">
 
+#### Workflow Actions
+
+> New in Version 11
+
+Workflow Actions facilitates communication to the users via email and also creates a list of pending Workflow Actions.
```

Comment:
Workflow Actions is a single place to manage all the pending actions you can take on Workflows.